worlds/alttp/__init__.py

```diff
@@ -296,6 +298,8 @@ def generate_early(self):
 
         player = self.player
         multiworld = self.multiworld
+        for option_name, option_value in self.options.as_dict():
```

Comment:
This doesn't work, as the old api expects the dict there, with entries per player